src/index.js

```diff
@@ -0,0 +1,164 @@
+  }
+};
+
+const landscapeChange = (temp) => {
+  let landscape = '';
+  if (parseInt(temp.textContent) >= 80) {
+    landscape = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (parseInt(temp.textContent) >= 70) {
+    landscape = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (parseInt(temp.textContent) >= 60) {
+    landscape = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (parseInt(temp.textContent) >= 50) {
+    landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (parseInt(temp.textContent) <= 49) {
+    landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+  return landscape;
+};
+
+const skyChangeOnSelect = () => {
+  const selectedSky = document.getElementById('sky-select').value;
+  const skyContainer = document.getElementById('sky-container');
+  const skyEmojiDisplay = skyChange(selectedSky);
+  skyContainer.textContent = skyEmojiDisplay;
+  console.log(document.getElementById('sky-select'));
+  console.log(skyEmojiDisplay);
+  console.log(selectedSky);
+};
+
+const skyChange = (skySelect) => {
+  let skyEmoji = '';
+  if (skySelect === 'sunny') {
+    skyEmoji = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+  } else if (skySelect === 'cloudy') {
+    skyEmoji = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (skySelect === 'rainy') {
+    skyEmoji = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (skySelect === 'snowy') {
+    skyEmoji = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  }
+  return skyEmoji;
+};
+
+const displayCityName = () => {
+  const inputCity = document.getElementById('city-name-input').value;
+  const displayCityContainer = document.getElementById('city-display');
+
+  displayCityContainer.textContent = '✨ ' + inputCity + ' ✨';
+};
+
+const resetCityName = () => {
+  const inputCity = document.getElementById('city-name-input');
+  const cityNameDisplay = document.getElementById('city-display');
+  inputCity.value = 'Seattle';
+  cityNameDisplay.textContent = 'Seattle';
+  console.log(inputCity);
+  console.log(inputCity.value);
+};
+
+// const response = get_location();
+// const long = response[0]['lat'];
+
+const registerEventHandlers = () => {
+  const increaseButton = document.getElementById('increase-temp');
+  const decreaseButton = document.getElementById('decrease-temp');
+  const cityInputForm = document.getElementById('city-name-input');
+  const selectedSky = document.getElementById('sky-select');
+  const realtimeTempButton = document.getElementById('realtime-temp');
+  const resetButton = document.getElementById('reset-city-name-2');
+  // const cityNameDisplay = document.getElementById('city-display');
+
+  increaseButton.addEventListener('click', increaseTemp);
+  decreaseButton.addEventListener('click', decreaseTemp);
+  cityInputForm.addEventListener('input', displayCityName);
+  selectedSky.addEventListener('change', skyChangeOnSelect);
+  realtimeTempButton.addEventListener('click', getTempFromCoordinates);
+  resetButton.addEventListener('click', resetCityName);
+  // cityNameDisplay.addEventListener('change', resetCityName);
+};
+
+document.addEventListener('DOMContentLoaded', registerEventHandlers);
```

Comment:
Nice job registering! Consider using the more robust form or registering that was at the bottom of the crab fan site example.